S21/hvkone/refactorme.py

```diff
@@ -2,83 +2,57 @@
 
 class Word:
 
-  def __init__(self, word, bcpCode, std=False):
-    self._w = word
-    self._l = bcpCode
-    self._finalSigma = False
-    self._standardIrishSpelling = std
-    # OLD EXPERIMENTAL CODE for dealing with vowel harmony
-    # self._numVowels = 0
-    # for c in word:
-    #   if c in 'aeiouAEIOU':
-    #   self._numVowels += 1
+  def __init__(self, word, language):
+    self.word = word 
+    self.lang = language
+  
+  def get_language(self):
```

Comment:
Might be nice to separate out this functionality into a class representing BCP-47 strings